akka-stream/src/main/scala/akka/stream/impl/CompletedPublishers.scala

```diff
@@ -40,6 +39,36 @@ private[akka] final case class ErrorPublisher(t: Throwable, name: String) extend
   override def toString: String = name
 }
 
+private[akka] final case class SingleElementPublisher[T](value: T, name: String) extends Publisher[T] {
+  import ReactiveStreamsCompliance._
+
+  private[this] class SingleElementSubscription(subscriber: Subscriber[_ >: T]) extends Subscription {
+    private[this] var done: Boolean = false
```

Comment:
yeah, only the initial value should be seen, since constructor and subscribe is called from a different thread than the one who calls request()